site/production-checklist.xml

```diff
@@ -0,0 +1,290 @@
+          access only to the users that will run RabbitMQ and tools such as <code>rabbitmqctl</code>.
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="security-considerations-tls">
+        <doc:heading>TLS</doc:heading>
+        <p>
+          We recommend using <a href="/ssl.html">TLS connections</a> when possible,
+          at least to encrypt traffic. Peer verification (authentication) is also recommended.
+          Development and QA environments can use <a href="https://github.com/michaelklishin/tls-gen/">self-signed TLS certificates</a>.
+          Self-signed certificates can be appropriate in production environments when
+          RabbitMQ and all applications run on a trusted network or iso.
+        </p>
+        <p>
+          While RabbitMQ tries to offer a secure TLS configuration by
+          default (e.g. SSLv3 is disabled), we recommend evaluating
+          what TLS versions and cipher suites are enabled.  Please see
+          our <a href="/ssl.html">TLS guide</a> for more information.
+        </p>
+      </doc:subsection>
+    </doc:section>
+
+
+    <doc:section name="Networking Configuration">
+      <doc:heading>Networking Configuration</doc:heading>
+
+      <doc:subsection name="networking-configuration-intro">
+        <p>
+          Production environments may require network configuration
+          tuning. Please refer to the <a href="/networking.html">Networking Guide</a> for details.
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="networking-configuration-automatic-recovery">
+        <doc:heading>Automatic Connection Recovery</doc:heading>
+        <p>
+          Some client libraries, for example, <a href="/api-guide.html">Java</a>, <a
+          href="/dotnet-api-guide.html">.NET</a>, and <a href="http://rubybunny.info">Ruby</a> ones, support
+          automatic connection recovery after network failures. If the
+          client used provides this feature, it is recommended to use
+          it instead of developing your own recovery mechanism.
+        </p>
+      </doc:subsection>
+    </doc:section>
+
+
+    <doc:section name="distribution-considerations">
+      <doc:heading>Clustering Considerations</doc:heading>
+
+      <doc:subsection name="distribution-considerations-cluster-size">
+        <doc:heading>Cluster Size</doc:heading>
+        <p>
+          When determining cluster size, it is important to take several factors into
+          consideration:
+
+          <ul>
+            <li>Expected throughput</li>
+            <li>Expected replication (number of mirrors)</li>
+            <li>Data locality</li>
+          </ul>
+        </p>
+        <p>
+          Since clients can connect to any node, RabbitMQ may need to perform inter-cluster
+          routing of messages and internal operations. Try making consumers and producers
+          connect to the same node, if possible: this will reduce inter-node traffic.
```

Comment:
If I do this, then why would I use a cluster besides HA? What about load balancing?